docs/UserGuide/Ecosystem Integration/MapReduce TsFile.md

```diff
@@ -34,7 +34,7 @@ With this connector, you can
 
 |Hadoop Version | Java Version | TsFile Version|
 |:---:|:---:|:---:|
-| `2.7.3`       | `1.8`        | `0.10.0+`|
+| `2.7.3`       | `1.8`        | `0.13.0-SNAPSHOT`|
```

Comment:
`0.10.0+` should be correct....

Reply:
0.10.0 could be in the doc of rel/0.11 branch, the master branch could keep up to the latest version?